### DISCUSSION TITLE
DOMException: Element.querySelectorAll: ':popover-open' is not a valid selector

### DISCUSSION THREAD

**Discussion by misog** [livewire/livewire#8941] [7. Bugs (archived)]
### Livewire version

v3.5.12

### Laravel version

v11.28.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Load any Livewire component

### Repository or Wirebox URL

-

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by daredloco** [comment]
One of our clients has the same problem on their machine. Login our our machines work just fine, but on theirs they can't do anything.
I can't reproduce it, so I can't give more details sadly but it is the first time since we use Livewire that we run into that problem.


Update: Client is using Windows XP, so there is that. Most likely caused to a totally outdated system as this did never happen to any other client of ours

**Reply by aleksanm** [reply]
I have the same issue in intranet with firefox 115.4.0esr and also on MS Edge the same.

**Comment by danie-ramdhani** [comment]
This maybe fixed in [this PR](https://github.com/livewire/livewire/pull/8972)

**Reply by aleksanm** [reply]
Noup. I updated to v3.5.12 and then this error shows up

**Reply by danie-ramdhani** [reply]
Because it's not tagged yet.

**Comment by aleksanm** [comment]
Thanks. I manualy modified on my production the code with this fix and it's works now.
